crates/relayer/src/chain/axon/msg.rs

```diff
@@ -288,15 +280,21 @@ impl From<contract::IdentifiedConnectionEndData> for IdentifiedConnectionEnd {
 
 impl From<contract::PacketData> for Packet {
     fn from(value: contract::PacketData) -> Self {
+        let timeout_height = if value.timeout_height.revision_height == 0 {
+            TimeoutHeight::Never
+        } else {
+            TimeoutHeight::At(value.timeout_height.into())
+        };
         Self {
             sequence: value.sequence.into(),
             source_port: value.source_port.as_str().parse().unwrap(),
             source_channel: value.source_channel.as_str().parse().unwrap(),
             destination_port: value.destination_port.as_str().parse().unwrap(),
             destination_channel: value.destination_channel.as_str().parse().unwrap(),
             data: value.data.as_ref().to_vec(),
-            timeout_height: TimeoutHeight::At(value.timeout_height.into()),
-            timeout_timestamp: Timestamp::from_nanoseconds(value.timeout_timestamp).unwrap(),
+            timeout_height,
+            timeout_timestamp: Timestamp::from_nanoseconds(value.timeout_timestamp * 100000)
```

Comment:
1 milliseconds (ms) is equal to 1000000 nanoseconds (ns). 

Why `100000` was used?

Reply:
that's my mistake